biomaj/bank.py

```diff
@@ -979,6 +980,10 @@ def remove_pending(self, release=None):
                 logging.debug("Remove:Pending:Dir:" + session.get_full_release_directory())
                 shutil.rmtree(session.get_full_release_directory())
             self.remove_session(pending['id'])
+            if last_update == pending_session_id:
+                self.banks.update({'name': self.name},
+                                  {'$set': {'last_update_session': ''}})
```

Comment:
I thought of it (unset) but wasn't sure if it was mandatory as parameter or not
I can update this also